spec/validators/tags_validator_spec.rb

```diff
@@ -19,8 +25,8 @@ class TagsValidatable
       model.tags = {
         organisations: { any: %w(dogs cats) },
         topics: { any: %w(dogs cats) },
-        foo: { any: %w(dogs cats) },
-        people: { any: %w(dogs cats) },
+        foo: { any: %w([dogs] !cats) },
+        people: { any: %w(\u0000) },
         world_locations: { any: "dogs" },
```

Comment:
💯 Nice work adding these. I actually think a couple more examples of valid and invalid tags would be helpful, since there are a lot of cases that we want to ensure we allow/disallow. E.g. a valid uuid like `f3bbdec2-0e62-4520-a7fd-6ffd5d36e03a` and an invalid string like `"><script>alert(1);</script>"` or something like that.